ops/global/main.tf

```diff
@@ -0,0 +1,73 @@
+locals {
+  team     = "dibbs"
+  project  = "ce"
+  env      = "global"
+  location = "eastus"
+
+  resource_group_name = "${local.team}-${local.project}-${local.env}"
+}
+
+#tfsec:ignore:azure-keyvault-specify-network-acl:exp:2024-05-01
+resource "azurerm_key_vault" "kv" {
+  name                        = "${local.team}${local.project}${local.env}kv"
+  location                    = local.location
+  resource_group_name         = local.resource_group_name
+  enabled_for_disk_encryption = true
+  tenant_id                   = data.azurerm_client_config.current.tenant_id
+  soft_delete_retention_days  = 7
+  purge_protection_enabled    = false
+
+  sku_name = "standard"
+}
+
+resource "azurerm_container_registry" "acr" {
+  location            = local.location
+  name                = "${local.team}${local.project}${local.env}acr"
+  resource_group_name = local.resource_group_name
+  sku                 = "Standard"
+  admin_enabled       = true
```

Comment:
Does admin need to be enabled? I'm curious what actions we'll need to perform that would require this to be `true`

Reply:
This section used the SimpleReport ACR as a model, with the same flag present. This will theoretically allow us to hand control to Octopus to leverage this resource to its fullest, if needed.

Reply:
Got it! 

For SR, I think it's enabled because we log in to the ACR from GH actions to push images. I'd imagine that if we did all the build/push image stuff in some Azure service, we could get away with this being false.